scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -59,6 +59,16 @@
                             order: Number.MIN_VALUE,
                             visibilityWindow: 300000000
                       },
+                      {
+                          name: "{{ clinvar_snvs.name }}",
+                          type: "{{ clinvar_snvs.type }}",
+                          format: "{{ clinvar_snvs.format }}",
+                          sourceType: "{{ clinvar_snvs.sourceType }}",
+                          url: "{{ clinvar_snvs.url|replace('%2F','/') }}",
+                          displayMode: "{{ clinvar_snvs.displayMode }}",
+                          order: Number.MIN_VALUE,
+                          visibilityWindow: 300000000
```

Comment:
Isn't that overdoing it a bit for snvs? Quite agree on the SV side though!

Reply:
Well, ok, it is quick anyway, so never mind!

Reply:
But in general, why not also the SV track?

Reply:
It appears it is called clinVarCnv.bb. It would be useful as well but feel free to leave that to another PR!

Reply:
I had it before and then I removed it, I'll reintroduce it!